pkg/metrics/metric_items.go

```diff
@@ -45,4 +45,20 @@ var (
 		Name: "task_node_replicate_object_task_number",
 		Help: "Track task node service replicate object task",
 	}, []string{serviceLabelName})
+	// PieceStoreTimeHistogram records piece store request time
+	PieceStoreTimeHistogram = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "piece_store_handling_seconds",
+		Help:    "Track the latency for piece store requests",
+		Buckets: prometheus.DefBuckets,
```

Comment:
What is the meaning of `seconds` here?

Reply:
The second level is not accurate enough?

Reply:
I use the same description as  https://github.com/grpc-ecosystem/go-grpc-middleware